qemu/tests/rng_bat.py

```diff
@@ -1,13 +1,13 @@
 import re
 import logging
 
-from autotest.client.shared import error
-from autotest.client.shared import utils
-
 from virttest import utils_misc
+from virttest import error_context
```

Comment:
Frankly speaking,
I don't think it's necessary that replace autotest.client.shared.error with virttest.error_context  *right now*.
Obviously, You support to replace the common function of autotest with avocado's or avocado-vt's.
In fact, If you use avocado-vt,  autotest is required (refer to avocado-vt's requirements.txt).
This case won't fail with autotest.client.shared.error , right?

Xutian told me that your team had shifted to avocado,
But  there're a lot of people who still use tp-qemu with autotest.
and recommended by the Qemu community 

One day in the feature, 
Once we definitely declare that autotest is not supported.
We can fix all the problems at once by using 'from virttest import error_context as error' instead of 'from autotest.client.shared import error' ,

Of course, If it must for you, I have no objection.